### ISSUE TITLE
Level: "Slot Machine"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#46]
Quite an easy level as there aren't that many options, but I thought it was a fun idea.
```
:name=Slot Machine
:description=Drop in a token and see where the rabbit ends up.
:num_rabbits=1
:num_to_save=1
:dig=1
:music=tryad-witness
         Q         
                   
#       /#\       #
#########p#########
 c     d #   #bd d 
 # i   # O   (## # 
   #b *  #  *  #b# 
    #(  c )    ### 
c #b#i  #  i i     
# b#b#c # i# #c   i
######## ##   #  (#
:*=b(
:*=i(
```

**Comment by colonelfazackerley** [comment]
Took me 5 goes. Thoroughly enjoyed it. Highly original.

**Comment by colonelfazackerley** [comment]
Close me! 
merged to master with f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a

**Comment by andybalaam** [comment]
Amazing! Again, excited at the new ideas!

**Comment by andybalaam** [comment]
First hint has an extra backslash.

**Comment by andybalaam** [comment]
Hints fixed in 1a1c3d8d8b835b03f06535233f19355b643252cc
